### ISSUE TITLE
`cols` no longer works in `@select`

### ISSUE THREAD

**Issue by CameronBieganek** [JuliaData/DataFramesMeta.jl#128]
`cols` no longer works in `@select`. It appears to still work in `@with`.

```julia
julia> using DataFrames

julia> using DataFramesMeta

julia> VERSION
v"1.1.0"

(v1.1) pkg> status DataFrames DataFramesMeta
    Status `~/.julia/environments/v1.1/Project.toml`
  [34da2185] Compat v2.1.0
  [a93c6f00] DataFrames v0.18.2
  [1313f7d8] DataFramesMeta v0.4.1
  [2913bbd2] StatsBase v0.30.0

julia> df = DataFrame(a = 1:3, b = 4:6)
3×2 DataFrame
│ Row │ a     │ b     │
│     │ Int64 │ Int64 │
├─────┼───────┼───────┤
│ 1   │ 1     │ 4     │
│ 2   │ 2     │ 5     │
│ 3   │ 3     │ 6     │

julia> colref = :b
:b

julia> @select(df, cols(colref))
ERROR: BoundsError: attempt to access 2-element Array{AbstractArray{T,1} where T,1} at index [[4, 5, 6]]
Stacktrace:
 [1] throw_boundserror(::Array{AbstractArray{T,1} where T,1}, ::Tuple{Array{Int64,1}}) at ./abstractarray.jl:484
 [2] checkbounds at ./abstractarray.jl:449 [inlined]
 [3] _getindex at ./multidimensional.jl:641 [inlined]
 [4] getindex at ./abstractarray.jl:927 [inlined]
 [5] getindex(::DataFrame, ::Array{Int64,1}) at /home/cbieganek/.julia/packages/DataFrames/m9gd9/src/dataframe/dataframe.jl:302
 [6] select at /home/cbieganek/.julia/packages/DataFramesMeta/Ykidf/src/DataFramesMeta.jl:292 [inlined]
 [7] (::getfield(Main, Symbol("###371#4")){DataFrame})(::Array{Int64,1}) at /home/cbieganek/.julia/packages/DataFramesMeta/Ykidf/src/DataFramesMeta.jl:71
 [8] top-level scope at /home/cbieganek/.julia/packages/DataFramesMeta/Ykidf/src/DataFramesMeta.jl:73

julia> @with(df, :a + cols(colref))
3-element Array{Int64,1}:
 5
 7
 9
```

System Info: Ubuntu 18.04

**Comment by nalimilan** [comment]
Since you say "no longer", do you know in what version it used to work?

**Comment by CameronBieganek** [comment]
Ah, good question. I _thought_ it used to work, but maybe I had been using `@where` instead of `@select`. `@where` does work:

```julia
julia> @where(df, cols(colref) .> 4)
2×2 DataFrame
│ Row │ a     │ b     │
│     │ Int64 │ Int64 │
├─────┼───────┼───────┤
│ 1   │ 2     │ 5     │
│ 2   │ 3     │ 6     │
```

The last time I was playing around with this was April 16, as evidenced by this Stack Overflow post:

https://stackoverflow.com/questions/55715518/column-name-as-a-variable-in-dataframesmeta-macros

**Comment by pdeffebach** [comment]
It doesn't look like `replace_syms` is ever called at any point in `@select`. Hopefully this is an easy fix.

**Comment by pdeffebach** [comment]
Fixed in #224
